packages/plugin-renderer-puppeteer/src/plugins/server.js

```diff
@@ -12,7 +12,7 @@ class PuppeteerServer extends ServerInterface {
       const { port } = this.compilation.config.devServer;
 
       (await getDevServer(this.compilation)).listen(port, async () => {
-        console.info(`Started puppeteer prerender server at localhost:${port}`);
+        console.info(`Started puppeteer prerender server at http://localhost:${port}`);
```

Comment:
This one probably won't ever get used by the user (just run at build time) but can't hurt!